projects/aca-content/src/lib/aca-content.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
not needed

Reply:
Hi Denys, it is a draft right now :) have you been still notify? My plan is to divide this PR to at least 2 commits.

One is to create `libraby` and everythig what is needed to make it work, second part is going to be moving the `content` to the library, so the number of moved files won't hide the changes